packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1224,6 +1224,47 @@ class EditableText extends StatefulWidget {
   ///   works only with [TextInputType.emailAddress]. Make sure the input field has a
   ///   compatible [keyboardType]. Empirically, [TextInputType.name] works well
   ///   with many autofill hints that are predefined on iOS.
+  ///
+  /// ### Troubleshooting Autofill
+  ///
+  /// Platform autofill services rely on [autofillHints] to work. Make sure the
+  /// entries in [autofillHints] are supported by the target platform, and the
+  /// autofill service currently in use (the name of the service can typically
```

Comment:
Nit: Do you mean "the autofill service is currently in use" ?

Reply:
I meant the hints need to be supported by both the target platform and the service (which was not accurate). Updated.

Reply:
Oh sorry, I misread it.  The new way looks good and isn't ambiguous 👍